### PR TITLE
finished cc13

### PR DESCRIPTION
**Pull request by ryanapodaca**

### CODE REVIEW

**Comment by ryanapodaca** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] README for this challenge is complete
       - [ ] Summary, Description, Approach & Efficiency, Solution
       - [ ] Picture of whiteboard
       - [ ] Link to code
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
       - [ ] “Happy Path” - Expected outcome
       - [ ] Expected failure
       - [ ] Edge Case (if applicable/obvious)

File: java/datastructures/lib/src/main/java/codechallenges/linkedlist/BracketLinter.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      Stack<String> stack = new Stack<>();
+
+      for (char c : s.toCharArray()) {
+        if (c == '{' || c == '[' || c == '(') {
+          stack.push(String.valueOf(c));
+        }
+
+        if (c == '}' && !stack.isEmpty() && stack.peek().equals("{")) {
+          stack.pop();
+          if (!stack.isEmpty() && !stack.peek().equals("{")) {
```
Comment (lucy-gelderloos) [inline java/datastructures/lib/src/main/java/codechallenges/linkedlist/BracketLinter.java:16]: I'm not sure what this if statement is doing - it looks like you've already removed the correct opening bracket, so if you check for it again, you're almost always going to get an error.
